javascript/src/ycell.ts

```diff
@@ -288,22 +292,6 @@ export class YBaseCell<Metadata extends nbformat.IBaseCellMetadata>
       : this.notebook.undoManager;
   }
 
-  /**
-   * Defer setting the undo manager as it requires the
-   * cell to be attached to the notebook Y document.
-   */
-  setUndoManager(): void {
```

Comment:
Should we remove it as it is not part of the public API defined in `api.ts` and was only used internally in `ynotebook.ts`, or should we keep it and deprecate, making it a no-op?

Reply:
If it's not part of the public API, I'd say we should remove it.